### PR TITLE
Azure CI

### PR DESCRIPTION
**Pull request by mdavis36**

### CODE REVIEW

**Comment by rhornung67** [comment]
@mdavis36 from the log files, it looks like azure is looking in the wrong place for test artifacts. Note that there are no tests to run in this project. We simply want to build and run via the command './bin/raja-perf.exe --checkrun -sp'. So the check would be (at least for now): runs to completion --> pass, does not run to completion --> fail.

**Review by rhornung67 — COMMENTED** [review]
There are no TBB kernel variants to run. So that should be left off. Also, we want to make sure that OpenMP is enabled for the standard builds. I believe that is the cause of most of the unused parameter 'vid' warnings. I will make a point to fix that.

File: Dockerfile [hunk 1]
```diff
@@ -10,55 +10,55 @@ ENV GTEST_COLOR=1
 COPY --chown=axom:axom . /home/axom/workspace
 WORKDIR /home/axom/workspace
 RUN ls
-RUN mkdir build && cd build && cmake -DCMAKE_CXX_COMPILER=g++ -DENABLE_WARNINGS=On -DENABLE_TBB=On -DRAJA_DEPRECATED_TESTS=On ..
+RUN mkdir build && cd build && cmake -DCMAKE_CXX_COMPILER=g++ -DENABLE_WARNINGS=On -DENABLE_OPENMP=On -DRAJA_DEPRECATED_TESTS=On ..
```
Comment (rhornung67) [inline Dockerfile:13]: 👍

File: Dockerfile [hunk 2]
```diff
@@ -12,7 +12,7 @@ WORKDIR /home/axom/workspace
 RUN ls
 RUN mkdir build && cd build && cmake -DCMAKE_CXX_COMPILER=g++ -DENABLE_WARNINGS=On -DENABLE_OPENMP=On -DRAJA_DEPRECATED_TESTS=On ..
 RUN cd build && make -j 16
-RUN cd build && ./bin/raja-perf.exe test --output-on-failure
+RUN cd build && ./bin/raja-perf.exe
```
Comment (rhornung67) [inline Dockerfile:15]: 👍

**Review by davidbeckingsale — APPROVED** [review]

**Review by rhornung67 — APPROVED** [review]

**Comment by mdavis36** [comment]
@rhornung67 This is coming back green now. Some things to note:
- It looks like the perfsuite can't build with msvc, this seems to be due to some of the included headers. For the time being I've commented out the windows azure CI jobs.
- I've turned off perf suite runs for the debug builds as they were taking over an hour to complete.
- Sycl CI can be activated in a later PR when it has been merged with RAJA.

**Comment by rhornung67** [comment]
@mdavis36 let's keep travis up along with azure for a while before we turn off travis.

**Comment by rhornung67** [comment]
Turning off msvc is fine. it's not a priority for this. we can enable it later is needed.

@mdavis36 can you change the run commands to ./bin/raja-perf.exe --checkrun -sp  (i.e., add the two args) for the debug builds? That should be good enough for CI testing and should run quickly.

**Pull request merged**